src/compiler/abstract_compiler.nit

```diff
@@ -1837,6 +1838,8 @@ redef class MClassType
 			return "char*"
 		else if mclass.name == "NativeArray" then
 			return "val*"
+		else if mclass.name == "UnicodeChar" then
+			return "uint32_t"
```

Comment:
Tears were shed and blood was spilled, but yeah, I guess I managed to pass the test.